src/rust/engine/Cargo.toml

```diff
@@ -86,7 +86,7 @@ log = "0.4"
 num_enum = "0.1.1"
 parking_lot = "0.6"
 process_execution = { path = "process_execution" }
-reqwest = "0.9"
+reqwest = { git = "https://github.com/pantsbuild/reqwest.git", rev = "983409edb6ba1693f38a6fc230dfcd34f9249649", features = ["native-tls-vendored"] }
```

Comment:
It would be good if this linked to an upstream PR to add this feature to reqwest so that we don't have to maintain the fork.

Reply:
...which looks like https://github.com/seanmonstar/reqwest/pull/391 ... thanks!